packages/performance/src/services/remote_config_service.ts

```diff
@@ -34,22 +34,14 @@ interface SecondaryConfig {
   logSource?: number;
   logEndPointUrl?: string;
   transportKey?: string;
-  shouldSendToFl?: boolean;
   tracesSamplingRate?: number;
   networkRequestsSamplingRate?: number;
 }
 
 // These values will be used if the remote config object is successfully
 // retrieved, but the template does not have these fields.
 const DEFAULT_CONFIGS: SecondaryConfig = {
```

Comment:
Thought: It is nice to remove NO_TEMPLATE_CONFIGS in `remote_config_service.ts`. And I believe there is opportunity to improve even more around https://github.com/firebase/firebase-js-sdk/blob/acc545fc266340f6bb1136f2f6c3b43d68ff574c/packages/performance/src/services/remote_config_service.ts#L164-L225

Reply:
There is a lot of refactor/cleanup that can be done to make this more thinner and efficient. But I think that would be outside the context of this PR. Lets evaluate that separately.